go-apps/meep-rnis/server/rnis.go

```diff
@@ -189,19 +196,17 @@ type PlmnInfoResp struct {
 	PlmnInfoList []PlmnInfo
 }
 
-//MEC011 section begin
 const serviceAppName = "RNI"
 const serviceAppVersion = "2.1.1"
 
 var serviceAppInstanceId string
 
-var appEnablementClientUrl string = "http://meep-app-enablement"
-var appEnablementSupport bool = true
+var appEnablementUrl string
+var appEnablementEnabled bool = false
```

Comment:
Please don't initialize `appEnablementEnabled` as it suggest that this is the default mode which is not true.

The default mode is set by the environment variable.

Reply:
Okay, I will initialize it in the Init() function where the environment variable is read instead.